test/router_SUITE.erl

```diff
@@ -2950,6 +2973,12 @@ rx_delay_change_during_session_test(Config) ->
 
     %% Get `new_rx_delay` in Metadata:
     router_device_worker:device_update(WorkerPid),
+    {ok, Device1} = router_device_cache:get(?CONSOLE_DEVICE_ID),
+    Metadata1 = router_device:metadata(Device1),
+    ?assertEqual(ExpectedRxDelay1, maps:get(rx_delay, Metadata1)),
+    ?assertEqual(rx_delay_established, maps:get(rx_delay_state, Metadata1)),
+    %% State machine has cleaned-up after itself:
+    ?assert(maps:is_key(new_rx_delay, Metadata1), false),
```

Comment:
Regular `?assert` the second argument is a comment. I think you're looking for `?assertNot` or `?assertEqual`?

Reply:
Good catch!  Thanks.  Test still passes with this change.  Fix will go in next commit.  (Still working on ADR test failures with Michael...)

Reply:
Fixed in commit feb5a6f, but other issues persist in this revised test.